Source/qol/stash.cpp

```diff
@@ -538,6 +537,21 @@ void StashStruct::RemoveStashItem(uint16_t iv)
 	Stash.dirty = true;
 }
 
+void StashStruct::SetPage(int newPage)
+{
+	page = clamp(newPage, 0, 49);

```

Comment:
I think it'd be good to extract the 49 to a separate variable for visibility and change it to 99 :)

Reply:
I generalized `CountStashPages` and used it here and for `AutoPlaceItemInStash`.
I would like to separate the change to 100 pages from this bugfix-pr.
But I'm not against more pages. 😁